docs/labwc-actions.5.scd

```diff
@@ -54,6 +54,10 @@ Actions are used in menus and keyboard/mouse bindings.
 *<action name="MoveTo" x="" y="" />*
 	Move to position (x, y)
 
+*<action name="MoveToCursor" />*
+	Move to be centered on cursor.
+	While preventing any part off the window from going off-screen.
```

Comment:
If the window is larger than the output in either dimension it will still be partly off screen,
so I'd suggest something like `Tries to prevent any part of the the window to go off-screen`.